Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +470,86 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire=0):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array,
+        'expiry': time_to_expire
+    }
+
+    response = http_request('POST', uri_suffix, data=json.dumps(payload))
+    return response
+
+
+def ban_ip_command():
+    ip_addresses_string = demisto.args()['ip_address']
+    ip_addresses_array = argToList(ip_addresses_string)
+    for ip_address in ip_addresses_array:
+        if not is_ip_valid(ip_address, True):
+            return_error('Error: invalid IP address sent as argument.')
```

Comment:
Wondering if we should send the error after processing all the IPs, this way you just get an error and stop the program

+

Add a blank line after

Reply:
Could you explain the first comment please? What do you feel could be improved here, and how?

As for the blank line - will fix